std/assembly/typedarray.ts

```diff
@@ -1260,3 +1304,34 @@ function REVERSE<TArray extends ArrayBufferView, T>(array: TArray): TArray {
   }
   return array;
 }
+
+// @ts-ignore: decorator
+@inline
+function WRAP<TArray extends ArrayBufferView, T>(buffer: ArrayBuffer, byteOffset: i32 = 0, length: i32 = -1): TArray {
+  if (byteOffset < 0 || byteOffset >= buffer.byteLength) {
```

Comment:
minor: looks like `buffer.byteLength` is a candidate for caching its value in a local because it is loaded at least twice

Reply:
Fixed